index.js

```diff
@@ -86,9 +87,9 @@ function Request(url, options, f, retryConfig) {
 
   /**
    * Return a number representing how long request-retry should wait before trying again the request
-   * @type {Boolean} (err, response, body) -> Number
+   * @type {Boolean} (err, response, body, attempts) -> Number
    */
-  this.delayStrategy = _.isFunction(options.delayStrategy) ? options.delayStrategy : function() { return this.retryDelay; };
+  this.delayStrategy = _.isFunction(options.delayStrategy) ? options.delayStrategy :  DelayStrategies.Exponential;
```

Comment:
hum this change is not backward compatible because exponential (by definition) does not take the retryDelay parameter. So the default value should be DelayStrategies.Linear.

And we should also mark retryDelay as deprecated I think